noir-projects/aztec-nr/aztec/src/utils/field.nr

```diff
@@ -0,0 +1,201 @@
+use std::option::Option;
+
+global KNOWN_NON_RESIDUE: Field = 5; // This is a non-residue in Noir's native Field.
+
+global C1: u32 = 28;
+global C3: Field = 40770029410420498293352137776570907027550720424234931066070132305055;
+global C5: Field = 19103219067921713944291392827692070036145651957329286315305642004821462161904;
```

Comment:
?!

Reply:
Yeah, this is bad. The good news is, it was like this in Noir's ec lib, so it's not my fault. The bad news is: it is my fault that I copied it here.